### PR TITLE
change to log.go v2

### PR DESCRIPTION
**Pull request by wavemechanics**
### What

Change to log.go v2

### How to review

make test

### Who can review

Anybody who is familiar with log.go v2.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -254,12 +254,12 @@ func (cli *S3) CheckPartUploaded(ctx context.Context, req *UploadPartRequest) (b
...
 	for _, part := range parts {
 		if *part.PartNumber == int64(req.ChunkNumber) {
-			log.Event(ctx, "chunk number already uploaded", log.INFO, log.Data{"chunk_number": req.ChunkNumber, "file_name": req.FileName, "identifier": req.UploadKey})
+			log.Info(ctx, "chunk number already uploaded", log.Data{"chunk_number": req.ChunkNumber, "file_name": req.FileName, "identifier": req.UploadKey})
 			return true, nil
 		}
 	}
 
-	log.Event(ctx, "chunk number failed to upload", log.ERROR, log.Data{"chunk_number": req.ChunkNumber, "file_name": req.FileName})
+	log.Error(ctx, "chunk number failed to upload", nil, log.Data{"chunk_number": req.ChunkNumber, "file_name": req.FileName})
```
Comment (cadmiumcat) [inline client.go]: I'm wondering if we're meant to always pass an error when using `log.Error` (instead of `nil`)?

**Review by cadmiumcat — COMMENTED** [review]
Tests pass and this looks good, just the one question 👍

File: client.go (hunk 1 above)
Comment (wavemechanics) [reply client.go]: Fixed to pass sensible error in those two cases. Thanks!

**Review by cadmiumcat — DISMISSED** [review]
LGTM 👍

**Review by cadmiumcat — DISMISSED** [review]

**Review by cadmiumcat — DISMISSED** [review]

**Review by cadmiumcat — APPROVED** [review]

**Pull request merged**